hack/copy-frontend-src-to-ossmc.sh

```diff
@@ -44,6 +45,7 @@ while [[ $# -gt 0 ]]; do
   key="$1"
   case $key in
     -dr|--dest-repo)     DEST_REPO="$2"         ;shift;shift ;;
+    -pr|--pull-request)  PULL_REQUEST="$2"      ;shift;shift ;;
```

Comment:
add a brief --help message down between lines 61 and 63 to let people know what this option is.